packages/cactus-test-geth-ledger/src/test/typescript/solidity/simple-token-contract/SimpleToken.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.5.1;
```

Comment:
@rwat17 Please use a newer compiler if possible something like this `pragma solidity 0.8.19;` that we have in the other contracts. Also important to pin the version down so that the hard to trace bugs are circumvented where the ABI gets broken with minor updates of the solidity compiler (I spent days debugging one such issue a few weeks back)

Reply:
i decided to reuse already compiled contract from other package :)

Reply:
@rwat17 Even better! Thank you!